client/src/components/Sidebar/SidebarContent.tsx

```diff
@@ -0,0 +1,35 @@
+import SidebarRoutes from "../../routes/SidebarRoutes";
+import {iSidebarRoute} from "../../types";
+import {Link, NavLink} from "react-router-dom";
+import {PlusCircleIcon} from "@heroicons/react/outline";
+
+//text-gray-800 dark:text-gray-100
+
+const SidebarContent = (): JSX.Element => {
+    return <div className="py-4 text-gray-500 dark:text-gray-400">
+        <Link to="/app/" className="ml-6 text-lg font-bold text-primary">
+            Jobssy
+        </Link>
+        <ul className="mt-6">
+            {SidebarRoutes.map((route: iSidebarRoute, idx: number) =>
+                <li className="relative px-6 py-3" key={idx}>
```

Comment:
Ah you should avoid using idx for keys IIRC. If SidebarRoutes get sorted into a different list, I don't think the updates would be captured.

It'd be better use something like `route.path` here instead.